src/pages/event-schema.ts

```diff
@@ -15,8 +15,20 @@ export const eventDateSchema = Joi.date()
     "date.max": `Date must be before ${maxDate.toISOString().split("T")[0]}`,
   });
 
-export const eventSchema = Joi.object({
-  eventDate: eventDateSchema,
+export const endDateSchema = Joi.date()
+  .iso()
+  .allow("")
+  .min(Joi.ref("startDate"))
+  .max(maxDate)
+  .messages({
+    "date.format": `Date must be a real date`,
+    "date.min": `The end date must be the same as or after the start date`,
+    "date.max": `Date must be before ${maxDate.toISOString().split("T")[0]}`,
+  });
+
+export const stageSchema = Joi.object({
```

Comment:
file also can be rename to "stage-schema"

Reply:
Jinx